known-plugins.json

```diff
@@ -9,6 +9,10 @@
       "hmrc-frontend",
       "jquery",
       "notifications-node-client"
+    ],
+    "mandatory": [
```

Comment:
I'm nervous about "mandatory" including `govuk-frontend`, it might be a naming thing.  I presume this is to separate out the ones that the user can't uninstall in the UI at the moment?

Reply:
Yes, that's correct

Reply:
That was one reason, but I think it was more general than that - that the user experience in general relies on Frontend - if it's missing things look and feel broken. It'd be good to review the user experience when Frontend is removed to see what it's like. Seems like that work would block this, so maybe lets leave 'mandatory' for now and remove when we're ready

Reply:
just thinking 'required' might be better than 'mandatory' - one to run by @oli-rose28

Reply:
I agree that "required" is a better name than "mandatory" so I have made the change as suggested.